.github/workflows/release.yml

```
@@ -0,0 +1,16 @@
+---
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: Release gem
+    uses: theforeman/actions/.github/workflows/release-gem.yml@v0
+    with:
+      allowed_owner: theforeman
```

```suggestion
      allowed_owner: Katello
```

Unless you wanted to migrate that repo over to @theforeman :)

---

Updated and changed the secret to the key you sent me in a PM